installer/resources/pacbot_app/files/DB.sql

```diff
@@ -1225,9 +1225,9 @@ INSERT IGNORE INTO `cf_Target` (`targetName`,`displayName`,`targetDesc`,`categor
 INSERT IGNORE INTO `cf_Target` (`targetName`,`displayName`,`targetDesc`,`category`,`dataSourceName`,`targetConfig`,`status`,`userId`,`endpoint`,`createdDate`,`modifiedDate`,`domain`) VALUES ('clouddns','Cloud DNS','GCP Cloud DNS','Google Cloud','gcp','{"key":"id","id":"id"}','enabled','admin',concat(@eshost,':',@esport,'/gcp_clouddns/clouddns'),'2022-06-01','2022-06-01','Infra & Platforms');
-INSERT IGNORE INTO `cf_Target` (`targetName`,`displayName`,`targetDesc`,`category`,`dataSourceName`,`targetConfig`,`status`,`userId`,`endpoint`,`createdDate`,`modifiedDate`,`domain`) VALUES ('cloudsql_sqlserver','SQL Server','GCP Cloud SQL Server','CloudSQLServer','gcp','{"key":"id","id":"id"}','enabled','admin',concat(@eshost,':',@esport,'/gcp_cloudsql_sqlserver'),'2022-06-29','2022-06-29','Infra & Platforms');
-INSERT IGNORE INTO `cf_Target` (`targetName`,`displayName`,`targetDesc`,`category`,`dataSourceName`,`targetConfig`,`status`,`userId`,`endpoint`,`createdDate`,`modifiedDate`,`domain`) VALUES ('cloudsql_mysqlserver','MySQL Server','GCP Cloud MySQL Server','CloudMySQLServer','gcp','{"key":"id","id":"id"}','enabled','admin',concat(@eshost,':',@esport,'/gcp_cloudsql_mysqlserver'),'2022-10-04','2022-10-04','Infra & Platforms');
-INSERT IGNORE INTO `cf_Target` (`targetName`,`displayName`,`targetDesc`,`category`,`dataSourceName`,`targetConfig`,`status`,`userId`,`endpoint`,`createdDate`,`modifiedDate`,`domain`) VALUES ('cloudsql_postgres','PostgreSQL','GCP Cloud postgres Server','CloudPostgresServer','gcp','{"key":"id","id":"id"}','enabled','admin',concat(@eshost,':',@esport,'/gcp_cloudsql_postgres'),'2022-10-07','2022-10-07','Infra & Platforms');
+INSERT IGNORE INTO `cf_Target` (`targetName`,`displayName`,`targetDesc`,`category`,`dataSourceName`,`targetConfig`,`status`,`userId`,`endpoint`,`createdDate`,`modifiedDate`,`domain`) VALUES ('cloudsql_sqlserver','SQL Server','GCP Cloud SQL Server','CloudSQLServer','gcp','{"key":"id,databaseVersion","id":"id"}','enabled','admin',concat(@eshost,':',@esport,'/gcp_cloudsql_sqlserver'),'2022-06-29','2022-06-29','Infra & Platforms');
+INSERT IGNORE INTO `cf_Target` (`targetName`,`displayName`,`targetDesc`,`category`,`dataSourceName`,`targetConfig`,`status`,`userId`,`endpoint`,`createdDate`,`modifiedDate`,`domain`) VALUES ('cloudsql_mysqlserver','MySQL Server','GCP Cloud MySQL Server','CloudMySQLServer','gcp','{"key":"id,databaseVersion","id":"id"}','enabled','admin',concat(@eshost,':',@esport,'/gcp_cloudsql_mysqlserver'),'2022-10-04','2022-10-04','Infra & Platforms');
+INSERT IGNORE INTO `cf_Target` (`targetName`,`displayName`,`targetDesc`,`category`,`dataSourceName`,`targetConfig`,`status`,`userId`,`endpoint`,`createdDate`,`modifiedDate`,`domain`) VALUES ('cloudsql_postgres','PostgreSQL','GCP Cloud postgres Server','CloudPostgresServer','gcp','{"key":"id,databaseVersion","id":"id"}','enabled','admin',concat(@eshost,':',@esport,'/gcp_cloudsql_postgres'),'2022-10-07','2022-10-07','Infra & Platforms');
```

Comment:
I think the insert scripts will be ignored since that target type is already present in the table and targetName is the primary key. I think we need to go for update statement

Reply:
Right, changes done.

Reply:
LGTM